gnovm/pkg/gnolang/preprocess.go

```diff
@@ -157,24 +157,19 @@ func initStaticBlocks(store Store, ctx BlockNode, bn BlockNode) {
 			switch n := n.(type) {
 			case *AssignStmt:
 				if n.Op == DEFINE {
-					var defined bool
 					for _, lx := range n.Lhs {
 						nx := lx.(*NameExpr)
 						ln := nx.Name
 						if ln == blankIdentifier {
 							continue
 						}
 						if !isLocallyDefined2(last, ln) {
```

Comment:
it should be ok if nothing defined here, cuz it may be a re-definition, which means the `name` already exists.